Source/driver/Castro.cpp

```diff
@@ -3281,7 +3281,7 @@ Castro::computeTemp(MultiFab& State, Real time, int ng)
 
       ca_compute_lap_term(BL_TO_FORTRAN_BOX(bx0),
                           BL_TO_FORTRAN_FAB(Stemp[mfi]),
-                          BL_TO_FORTRAN_ANYD(Eint_lap[mfi]), &Eint,
+                          BL_TO_FORTRAN_ANYD(Eint_lap[mfi]), UEINT,
```

Comment:
I think you need to keep the reference here, to make the Fortran interface work. Same for the next change.